composer.json

```diff
@@ -20,7 +20,7 @@
         "laravel/telescope": "^5.0",
         "laravel/tinker": "^2.9",
         "maatwebsite/excel": "^3.1",
-        "phpoffice/phpword": "1.2",
+        "phpoffice/phpword": "1.1",
```

Comment:
What is the problem with `1.2` version? In the releases I don't see a lot of breaking changes:
https://github.com/PHPOffice/PHPWord/releases

Reply:
It doesn't work as expected. For some reason new version of package creates files in `/tmp` with some random names, but doesn't remember these names and returns something different (and non-existing). We should study it and try to solve it, but maybe not in this PR.

Reply:
Sure. Created issue for that:
- https://github.com/blumilksoftware/toby/issues/420